.github/workflows/contract-testing.yml

```diff
@@ -0,0 +1,64 @@
+name: 'Contract Testing'
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  change-detection:
+   runs-on: ubuntu-latest
+   permissions:
+     pull-requests: read
+     repository-projects: read
+   outputs:
+     project: ${{ steps.filter.outputs.project }}
+   steps:
+   - uses: dorny/paths-filter@v2
+     id: filter
+     with:
+      filters: |
+        project:
+          - 'cfn-resources/project/**'
+  project:
```

Comment:
Is the plan to keep all resources in this file then?

Reply:
yes, I think is the best solution since they will all depend on the path filter job. The other option is to create a file per resource.

Reply:
yes, I think is the best solution since they will all depend on the path filter job. The other option is to create a file per resource.